system/ee/ExpressionEngine/Addons/spam/View/modal.php

```diff
@@ -2,7 +2,7 @@
 	<div class="modal">
 		<div class="col-group">
 			<div class="col w-16">
-				<a class="m-close" href="#"></a>
+				<a class="m-close" href="#"><span class="sr-only">Close button</span></a>
```

Comment:
I think we need to use lang file here

Reply:
moved all constant text to the lang files